content/05.heterogeneity.md

```diff
@@ -1,22 +1,22 @@
+Other methods like multidimensional scaling (MDS), t-distributed stochastic neighbor embedding (t-SNE), and uniform manifold approximation and projection (UMAP) can also help identify useful patterns in the data, though t-SNE and UMAP may require adjusting the hyperparameters that may affect the output [@https://arxiv.org/abs/1802.03426; @doi:10.23915/distill.00002].
 Testing multiple dimensionality reduction methods, rather than a single method, may be necessary to obtain a more comprehensive portrait of the data [@doi:10.1186/s13059-020-02021-3]. 
 Nguyen and Holmes discuss additional important considerations for using dimensionality reduction methods such as selection criteria and interpretation of results [@doi:10.1371/journal.pcbi.1006907].
 Beyond dimensionality reduction, other unsupervised learning approaches such as k-means clustering or hierarchical clustering have been used to characterize the structure present in genomic or imaging data [@doi:10.1186/1471-2105-9-497; @doi:10.1109/jbhi.2013.2276766].
 
-Representation learning approaches, of which dimensionality reduction methods are a subset, learns low-dimensional representations (composite features) from the raw data. 
-For example, representation learning through matrix factorization can extract features from transcriptomics datasets that are made of combinations of gene expression values found in the training data [@doi:10.1038/s41467-020-14666-6], and use them to interpret test data [@doi:10.1093/bioinformatics/btq503; @doi:10.1186/s13059-020-02021-3].
-To ensure that the learned representations are generalizable to other data, the features learned by the model can be constrained through methods like regularization [@doi:10.1371/journal.pgen.1004754, @doi:10.1002/sim.6782]. 
+Dimensionality reduction methods are a subset of representation learning, which learns low-dimensional representations (or composite features) from the raw data. 
+For example, representation learning through matrix factorization method can extract features from transcriptomics datasets that are made of combinations of gene expression values [@doi:10.1038/s41467-020-14666-6; @doi:10.1093/bioinformatics/btq503; @doi:10.1186/s13059-020-02021-3].
 Representation learning generally requires many samples when applied to complex biological systems and therefore may appear to aggravate the curse of dimensionality. 
 However, it can be a powerful tool to learn low-dimensional patterns from large datasets and then find those patterns in smaller, related datasets. 
 In later sections, we will discuss this method of leveraging large datasets to reduce dimensionality in smaller datasets, also known as feature-representation-transfer learning. 
+Once the dimensions of the training dataset have been reduced, model training can proceed using the experimental design as outlined in Box 1.
```

Comment:
Is this supposed to speak to the context of feature-representation-transfer or be more general?

Reply:
I think more general